nav2_behavior_tree/nav2_tree_nodes.xml

```diff
@@ -247,6 +247,23 @@
       <input_port name="server_timeout"> Server timeout </input_port>
     </Condition>
 
+    <Condition ID="WouldAControllerRecoveryHelp">
+      <input_port name="error_code">Error code</input_port>
+    </Condition>
+
+    <Condition ID="WouldAPlannerRecoveryHelp">
+      <input_port name="error_code">Error code</input_port>
+    </Condition>
+
+    <Condition ID="WouldASmootherRecoveryHelp">
+      <input_port name="error_code">Error code</input_port>
+    </Condition>
+
+    <Condition ID="AreErrorCodesPresent">
```

Comment:
Missing the input port of `error_codes_to_check`

Reply:
This is not used by the other 3, but this is the base class that allows users to use it on its own so they can specify arbitrary numbers